src/screens/WalletDetailsScreen.tsx

```diff
@@ -137,6 +137,8 @@ export class WalletDetailsScreen extends React.PureComponent<Props> {
       return;
     }
     navigation.navigate(Route.EditText, {
+      inputTestID: 'wallet-name-input',
```

Comment:
`inputTestID` and `submitButtonTestID` seems to be unused

Reply:
It is used - Dawid changed `EditText` component in the previous commit
```
  return (
    <ScreenTemplate
      footer={
        <Button
          testID={submitButtonTestID}
          title={i18n._.save}
          onPress={handlePressOnSaveButton}
          disabled={!canSubmit()}
        />
      }
      header={<Header isBackArrow={true} title={title} />}
    >
      {header}
      <View style={styles.inputItemContainer}>
        <InputItem
          testID={inputTestID}
          label={label}
          value={checkZero ? checkZero(value) : value}
          setValue={setValue}
          autoFocus={true}
          error={error || (value && !!validate && validate(value)) || ''}
          keyboardType={keyboardType}
          maxLength={maxLength}
        />
      </View>
    </ScreenTemplate>
  );
};```